docs/types/operators.rst

```diff
@@ -26,6 +26,23 @@ except for comparison operators where the result is always ``bool``.
 The operators ``**`` (exponentiation), ``<<``  and ``>>`` use the type of the
 left operand for the operation and the result.
 
+Ternary Operator
+----------------
+The ternary operator is used in expressions of the form ``<expression> ? <trueExpression> : <falseExpression>``.
+It executes one of the two given expressions depending upon the result of the evaluation of the main ``expression``.
+If ``expression`` evaluates to True, then ``<trueExpression>`` will be executed, else, ``<falseExpression>`` is executed.
```

Comment:
```suggestion
It evaluates one of the latter two given expressions depending upon the result of the evaluation of the main ``expression``.
If ``expression`` evaluates to ``true``, then ``<trueExpression>`` will be evaluated, otherwise ``<falseExpression>`` is evaluated.
```